### PR TITLE
To fetch 3 messages from the java servlet

### PR DESCRIPTION
**Pull request by AbakirH**
Added an arrayList to the java file and converting to JSON data

Added a second button on my HTML page to fetch the JSON data and append it to the HTML page.
http://abakir-step-2020.ue.r.appspot.com/

### CODE REVIEW

**Review by nightrunner500 — COMMENTED** [review]
Can you make the title more descriptive, outlining the goal for this pull request?

File: portfolio/pom.xml [hunk 1]
```diff
@@ -37,6 +42,7 @@
           <!-- TODO: set project ID. -->
           <deploy.projectId>abakir-step-2020</deploy.projectId>
           <deploy.version>1</deploy.version>
+          <automaticRestart>true</automaticRestart>
```
Comment (nightrunner500) [inline portfolio/pom.xml]: This doesn't seem to align with the scope of this pull request.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -40,7 +40,26 @@ function viewGame(){
...
+function getJSONData(){
+  fetch('/data')  
+  .then(response => response.json()) 
+  .then((message) => {
+
+      let messageConntainer = document.getElementById("messages");
+      console.log(message[0]);
+
+      for(let i = 0; i < message.length; i++){
+        let text = document.createTextNode(message[i]);
```
Comment (nightrunner500) [inline portfolio/src/main/webapp/script.js:59]: Do you want to attach this to a standard html tag: div, p?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -40,7 +40,26 @@ function viewGame(){
...
   });
+}
+
+function getJSONData(){
+  fetch('/data')  
+  .then(response => response.json()) 
+  .then((message) => {
+
+      let messageConntainer = document.getElementById("messages");
+      console.log(message[0]);
```
Comment (nightrunner500) [inline portfolio/src/main/webapp/script.js]: Is there a useful reason for logging this? If so, please add a comment. If not, please remove.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (AbakirH) [reply portfolio/src/main/webapp/script.js:59]: I did not specify where the text will go, but I will make it so it goes in a p tag in my next commit

File: portfolio/pom.xml (hunk 1 above)
Comment (AbakirH) [reply portfolio/pom.xml]: I was trying to find a way when I refresh the page, my updated code shows, but I am still working on that

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (AbakirH) [reply portfolio/src/main/webapp/script.js]: I had it, for debugging purposes, but I forgot to delete it

File: portfolio/pom.xml (hunk 1 above)
Comment (nightrunner500) [reply portfolio/pom.xml]: This is still outside of the scope for this change. I'd suggest moving it to a different pull request if you want to commit it to this remote repository.

Comment (AbakirH) [reply portfolio/pom.xml]: okay I will do that next time

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (nightrunner500) [reply portfolio/src/main/webapp/script.js:59]: Thank you.

File: portfolio/pom.xml (hunk 1 above)
Comment (nightrunner500) [reply portfolio/pom.xml]: Ok. Just send another review once you've removed it from here, so I can take another look.

Comment (AbakirH) [reply portfolio/pom.xml]: Okay just fixed it

Comment (AbakirH) [reply portfolio/pom.xml]: okay just fixed it

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -23,11 +26,23 @@
...
-  
+  private ArrayList<String> messages = new ArrayList<String>();
+
+  @Override
+  public void init() {
+    messages.add("Hello, how are you?");
+    messages.add("Hello, what is your name?");
+    messages.add("Hello, what is your age?");
+  }
   @Override
```
Comment (nightrunner500) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: Please add a space above this method.
ref: https://google.github.io/styleguide/javaguide.html

**Review by nightrunner500 — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (AbakirH) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: I am confused about what you mean, by adding a space?

Comment (AbakirH) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: I think I did it

Comment (AbakirH) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: I think I did it

Comment (nightrunner500) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:38]: Yes. Thank you.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -23,11 +26,24 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = "Abakir";
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello " + name +"!");
+    
+    response.getWriter().println(convertToJsonUsingGson(messages));
+  }
+  private String convertToJsonUsingGson(ArrayList<String> temp) {
```
Comment (nightrunner500) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Please add a space above this method.

**Review by nightrunner500 — APPROVED** [review]

**Pull request merged**